src/js/common/schemaform/save-load-actions.js

```diff
@@ -1,4 +1,7 @@
+import moment from 'moment';
+import Raven from 'raven-js';
 import environment from '../helpers/environment.js';
+import 'isomorphic-fetch';
```

Comment:
Because I'm checking if something is an instance of `Response`, and this adds that (and `Request`) as globals that I can use in the tests.